This was done to illustrate error handling when making system calls in linux.

In the case of doing it with C we use the errno.h library to catch the error and display it.

In the case of doing it in Python we don't have to explicitly write error handling.
